README.md

```diff
@@ -16,17 +16,17 @@
 ## Resources
 
 - [Documentation](https://redis.uptrace.dev)
-- [Discussions](https://github.com/go-redis/redis/discussions)
+- [Discussions](https://github.com/redis/go-redis/discussions)
 - [Chat](https://discord.gg/rWtp5Aj)
-- [Reference](https://pkg.go.dev/github.com/go-redis/redis/v9)
-- [Examples](https://pkg.go.dev/github.com/go-redis/redis/v9#pkg-examples)
+- [Reference](https://pkg.go.dev/github.com/redis/go-redis/v9)
+- [Examples](https://pkg.go.dev/github.com/redis/go-redis/v9#pkg-examples)
 
 ## Ecosystem
 
-- [Redis Mock](https://github.com/go-redis/redismock)
+- [Redis Mock](https://github.com/redis/go-redismock)
```

Comment:
redismock should stay in go-redis, right?

Reply:
We certainly think so.. though I'll bet we could help the community here too. I've at least added it to the plate to track (internally).

Reply:
Yeah, this should be reverted.